MyCryptoHQ/MyCrypto
# Paper Wallet Refactor

Relies on #1939.

### Description

Refactors the paper wallet component to save itself as a PNG, rather than the hacky popup window -> print behavior. This allows it to work consistently between web and app, allows us to remove the limited CSS-in-JS we had to make it work, and should make this facet of the app more robust overall.

### Changes

* Remove paper wallet printing, replace with html2canvas -> png of the wallet and save that image
* Change some of the copy to say "Save" instead of "Print" since you don't _have_ to print it.
* Remove CSS in JS from paper wallet & identicon.

### Steps to Test

1. Create a keystore file
2. Ensure paper wallet shows up fine
3. Ensure that clicking on "Save Paper Wallet" downloads it
4. Unlock the keystore file on Send, go to Wallet Info tab
5. Ensure that clicking on Save Paper Wallet opens a modal that allows you to do 3 & 4.
6. Do all of the above in the Electron app.
